.eslintrc.js

```diff
@@ -44,6 +42,13 @@ module.exports = {
     es2022: true,
   },
   overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: {
+        '@typescript-eslint/no-explicit-any': 'off',
+        '@typescript-eslint/consistent-type-imports': 'error',
+      },
+    },
```

Comment:
I moved these from the generic "rules" above because otherwise there was errors related to passing config through the babel parser to the ts-eslint parser

Reply:
Do we care about .mts? No biggie either way

Reply:
Yeah good spot! I noticed that in some of the tinkering I did earlier today outside this PR. I'll make sure to add this in some follow up work.